src/cljc/orcpub/dnd/e5/magic_items.cljc

```diff
@@ -333,6 +334,25 @@ Additionally, you can focus your senses as an action to magically discern the di
 direction to the closest dragon within 30 miles of you that is of the same type as the armor. This special action can’t be used again until the next dawn."
                      )})
 
+(defn
```

Comment:
There already is a `spell-attack-modifier-bonus` and a `spell-save-dc-bonus` in modifiers.cljc
Do they not provide the functionality your after?

They can be used like this:
```
::modifiers [(mod5e/spell-attack-modifier-bonus 2)
(mod5e/spell-save-dc-bonus 3)]
```